targets/ChibiOS/_nf-overlay/os/hal/ports/RP/LLD/RNGv1/rng_lld.c

```
@@ -117,6 +117,80 @@ void rng_lld_stop(void)
+            osalThreadSleepMilliseconds(1);
+        }
+
+        return false;
+
+    next_chunk:;
+    }
+
+    return true;
+
+#endif
+}
```

Cppcheck (2.21.0)

[error] Found an exit path from function with non-void return type that has missing return statement

(missingReturn)